lib/class-wp-rest-menus-controller.php

```diff
@@ -431,6 +480,62 @@ public function delete_item( $request ) {
 		return $response;
 	}
 
+	/**
+	 * Returns the value of a menu's auto_add
+	 *
+	 * @param int $menu_id The menu id to update the location form.
+	 *
+	 * @return Boolean The value of auto_add.
+	 */
+	function get_menu_auto_add( $menu_id ) {
+		$nav_menu_option = (array) get_option( 'nav_menu_options', array( 'auto_add' => array() ) );
+		if ( ! in_array( $menu_id, $nav_menu_option['auto_add'], true ) ) {
+			return false;
+		}
+		return true;
+	}
+
+	/**
+	 * Updates the menu's auto add from a REST request.
+	 *
+	 * @param int             $menu_id The menu id to update the location form.
+	 * @param WP_REST_Request $request The request object with menu and locations data.
+	 *
+	 * @return Boolean|Null   Return the value of auto_add or null if the auto_add
+	 *                        is not present in the request.
+	 */
+	function handle_auto_add( $menu_id, $request ) {
+		if ( ! isset( $request['auto_add'] ) ) {
+			return true;
+		}
+
+		$nav_menu_option = (array) get_option( 'nav_menu_options', array( 'auto_add' => array() ) );
+
+		if ( ! isset( $nav_menu_option['auto_add'] ) ) {
+			$nav_menu_option['auto_add'] = array();
+		}
+
+		$auto_add = (bool) $request['auto_add'];
```

Comment:
```suggestion
		$auto_add = $request['auto_add'];
```

The cast is unnecessary since the property is typed as a `boolean` in the schema.